java/configuration/src/main/java/sleeper/configuration/properties/instance/CompactionProperty.java

```diff
@@ -119,6 +119,16 @@ public interface CompactionProperty {
             .description("The maximum number of concurrent compaction tasks to run.")
             .defaultValue("300")
             .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.compaction.job.creation.concurrency.reserved")
+            .description("The reserved concurrency for the compaction lambda.\n " +
+                    "See reserved concurrency overview at: \r\n https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html\r\n")
+            .defaultValue(null)
+            .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_LAMBDA_CONCURRENCY_MAXIMUM = Index.propertyBuilder("sleeper.compaction.job.creation.concurrency.max")
+            .description("The maximum given concurrency allowed for the compaction lambda.\n " +
+                    "See maximum concurrency overview at: \r\n https://aws.amazon.com/blogs/compute/introducing-maximum-concurrency-of-aws-lambda-functions-when-using-amazon-sqs-as-an-event-source/")
```

Comment:
These properties talk about "the compaction lambda" in the description, and have names starting with "COMPACTION_LAMBDA_".

Please can those match the language used for the other compaction job creation lambda properties?

Please can we move these properties next to the other compaction job creation lambda ones?

Reply:
Addressed